1-starter-app/app.js

```diff
@@ -0,0 +1,126 @@
+'use strict'
```

Comment:
It would be nice if you can install prettier as a dev dependency and have it fix the formatting of `js` files.